File: app/src/main/java/ru/vegax/xavier/a3test/user_data/Geo.java
package ru.vegax.xavier.a3test.user_data;

public class Geo {


    private final String mLat;
    private final String mLng;

    public Geo(String lat, String lng) {
        mLat = lat;
        mLng = lng;
    }
    public String getLat() {
        return mLat;
    }

    public String getLng() {
        return mLng;
    }
}

File: app/src/main/java/ru/vegax/xavier/a3test/models/Geo.kt
package ru.vegax.xavier.a3test.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Geo {

    @SerializedName("lat")
    @Expose
    var lat: String? = null
    @SerializedName("lng")
    @Expose
    var lng: String? = null

}
